servicenow/plugin.spec.yaml

```
@@ -401,3 +401,27 @@ triggers:
         description: JSON object representing changed fields (map of field name to previous and current values)
         type: object
         required: true
+  incident_created:
+    title: Incident Created
+    description: Identifies if a new incident has been created
+    input:
+      query:
+        title: Query
+        description: Non-encoded query string to match incident records
+        type: string
+        required: false
+        example: short_description=Newbug
+      interval:
+        title: Interval
+        description: How often to detect new incidents (in seconds)
+        type: integer
+        required: false
```

```suggestion
        required: true
```
It's easier just to require it